Correct dependenies for .meta file generation

These targets require "chibi-scheme" binary to be present, but do not
specify this dependency explicitly. This breaks parallel builds (-jN)
and Debian build system uses that by default.

--- a/Makefile
+++ b/Makefile
@@ -208,7 +208,7 @@
 %.html: %.scrbl $(CHIBI_DOC_DEPENDENCIES)
 	$(CHIBI_DOC) --html $< > $@
 
-lib/.%.meta: lib/%/ tools/generate-install-meta.scm
+lib/.%.meta: lib/%/ tools/generate-install-meta.scm $(CHIBI_DEPENDENCIES) all-libs
 	-$(FIND) $< -name \*.sld | \
 	 $(CHIBI) tools/generate-install-meta.scm $(VERSION) > $@
 
